tdxhydrorapid/inputs.py

```
@@ -235,47 +454,73 @@ def rapid_master_files(streams_gpq: str,
     sgdf.drop(columns=['geometry', ]).to_parquet(os.path.join(save_dir, "rapid_inputs_master.parquet"))
     return
 
+def create_nexus_points(gdf: gpd.GeoDataFrame, 
+                        nexus_file: str, 
+                        id_field: str = 'LINKNO', 
+                        ds_id_field: str = 'DSLINKNO') -> None:
+    nexus_list = []
+    for river_id in gdf[gdf['LengthGeodesicMeters'] <= 0.01][id_field].values:
+        feat = gdf[gdf[id_field] == river_id]
+        upstreams = set(gdf[gdf[ds_id_field] == river_id][id_field])
+        if not feat[ds_id_field].values != -1 and all([x != -1 for x in upstreams]):
```

Ruff (0.8.0)

Use `feat[ds_id_field].values == -1` instead of `not feat[ds_id_field].values != -1`

Replace with `==` operator

(SIM202)